public/app/features/canvas/runtime/scene.tsx

```diff
@@ -332,6 +347,7 @@ export class Scene {
       })
       .on('dragStart', (event) => {
         this.ignoreDataUpdate = true;
+        this.disableNonTargets(event.target, true);
```

Comment:
I think this solution is ok given we are only firing this once on the beginning of drag event vs for every drag event 👍